Makefile

```diff
@@ -607,6 +609,25 @@ endif
 	@echo -n "pythemis install "
 	@$(BUILD_CMD_)
 
+pythemis_make_wheel: CMD = cd src/wrappers/themis/python/ && python3 setup.py bdist_wheel
```

Comment:
last comment. can we clean generated artifacts with `clean` target? this command creates new folder with source codes and wheel. or if we can specify destination folder, let's use `$(BIN_PATH)`. It's already mentioned in the `clean` target